app/models/retailer.rb

```diff
@@ -6,6 +6,7 @@ class Retailer < ApplicationRecord
 
   has_many :tills
   has_many :items
+  has_many :vouchers
```

Comment:
just had the has_many vouchers to retailer for the vouchers to be accessible through `retailer.vouchers`